### PR TITLE
Arc experimental v4

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
               "no-var-requires": false,
               "prefer-const": false,
               "quotemark": [true, "single"],
-              "prefer-for-of": false
+              "prefer-for-of": false,
+              "triple-equals": false
```
Comment (orenyodfat) [inline tslint.json]: why ?

Comment (orenyodfat) [reply tslint.json]: https://www.impressivewebs.com/why-use-triple-equals-javascipt/

File: package.json [hunk 2]
```diff
@@ -25,13 +25,14 @@
     "docker:rebuild": "docker-compose build",
```
Comment (orenyodfat) [inline package.json]: bump version ?

File: tslint.json (hunk 1 above)
Comment (ben-kaufman) [reply tslint.json]: This is part of a patch from a commit made in master to address the following issue:
https://github.com/daostack/subgraph/pull/473

File: package.json (hunk 2 above)
Comment (ben-kaufman) [reply package.json]: done

**Pull request merged**